zio-spark-core/src/main/scala/zio/spark/experimental/ZIOSparkAppDefault.scala

```diff
@@ -0,0 +1,9 @@
+package zio.spark.experimental
+
+import zio._
+
+trait ZIOSparkAppDefault extends ZIOSparkApp {
+  type Environment = Any
+
+  val bootstrap: ZLayer[ZIOAppArgs, Any, Any] = ZLayer.empty
```

Comment:
maybe, we should get the `sparkSession` from config directly. So the `ZIOSparkAppDefault` can be used directly.

Reply:
That might be good. In my app, I don't want to start the SparkSession on launch, because there are some commands (like `--help`) that don't need it. Perhaps that would be cause for another trait, e.g. `ZIOSparkSessionConfig`